chromium/rules.js

```diff
@@ -255,7 +255,7 @@ RuleSets.prototype = {
     }
 
     // If a ruleset declares a platform, and we don't match it, treat it as
-    // off-by-default. In practice, this excludes "mixedcontent" & "cacert" rules.
+    // off-by-default. In practice, this excludes "mixedcontent" rules.
```

Comment:
There is another reference to this in https://github.com/EFForg/https-everywhere/pull/14875/files#diff-8809792043b9914a5eea37df1e10a564R463